src/back/auth/src/model/session_refresh.hpp

```diff
@@ -0,0 +1,11 @@
+#pragma once
+
+#include <string>
+
+namespace svetit::auth::model {
+
+struct SessionRefresh {
+	std::string _token;
+};
+
+}
```

Comment:
Для того чтобы понимать что эта закрывающая скобка закрывает namespace, я предлагаю добавлять к ней комментарий. Она особенная по тому что на одном уровне с содержимым. В противном случае всё содержимое внутри нужно было бы оформить отступом.
```suggestion
} // namespace svetit::auth::model
```

Reply:
Поправил.